main.go

```diff
@@ -132,7 +142,7 @@ func mainE() error {
 			`%cn`: "GIT_CLONE_COMMIT_COMMITER_NAME",
 			`%ce`: "GIT_CLONE_COMMIT_COMMITER_EMAIL",
 		} {
-			if err := printLogAndExportEnv(gitCmd, format, env); err != nil {
+			if err := printLogAndExportEnv(gitCmd, format, env, (env == "GIT_CLONE_COMMIT_MESSAGE_SUBJECT" || env == "GIT_CLONE_COMMIT_MESSAGE_BODY")); err != nil {
```

Comment:
What's the motivation behind this change, actually? And how can I override this behavior?

(The real "question behind the question" is: why are my Slack notifications now showing truncated commit messages, and how can I get the full ones back? Please let me know if I should open a separate issue, or ask on the forum.)

Thanks!

Reply:
This has been updated, now it is limited by the maximum item size in envman. From one side it is much more than the 72 chars, on the other side, you can configure it by increasing envman item max size.
The motivation behind was that, some commit message or subject was extremely long, and the build failed, as envman was unable to save them, because they were longer than the limit.

Reply:
Thanks for the explanation and the update!